projects/packages/my-jetpack/changelog/update-php-requirements

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
This one could go two ways.

On the one hand we're only changing some tests, no change yet to the package itself. So in that sense it should be a patch bump, and no real need for a change entry.

On the other hand, possibly we should do a **major** bump for everything in `projects/packages/` to call out that we're no longer supporting PHP 5.6, even if there are no immediate breaking changes to any particular package.

Reply:
👍 I switched to a major version bump in 6caa55c72efd1f4cd8352fb39b30055590d6efbc